src/Stryker.Core/Stryker.Core/Initialisation/ProjectFileReader.cs

```diff
@@ -1,147 +1,145 @@
+namespace Stryker.Core.Initialisation;
 using System.Collections.Generic;
 using System.Linq;
-using Buildalyzer;
+using global::Buildalyzer;
```

Comment:
I've seen this in some places specifically for Buildalyzer, why?

Reply:
So, looks like a side-effect of the namespace/using moving. I'd need to read exactly why, but removing it makes the exposed types of that namespace unavailable.

I'll hold fire on this and dig some more. It might be that moving the usings inside the namespace is a different PR :-)